Update front matter, acknowledgements and criticality for AAD (#472)
cisagov/ScubaGear: baselines/aad.md

# CISA M365 Security Configuration Baseline for Azure Active Directory

> Azure Active Directory (AAD) is a cloud-based identity and access control service that provides security and functional capabilities to Microsoft 365. his Secure Configuration Baseline (SCB) provides specific policies to help secure AAD. 

Reviewer: ```suggestion
Azure Active Directory (AAD) is a cloud-based identity and access control service that provides security and functional capabilities to Microsoft 365. This Secure Configuration Baseline (SCB) provides specific policies to help secure AAD. 
```

Azure Active Directory (AAD) is a cloud-based identity and access control service that provides security and functional capabilities to Microsoft 365. This security baseline provides policies to help secure AAD. 

The Secure Cloud Business Applications (SCuBA) project run by the Cybersecurity and Infrastructure Security Agency (CISA) provides guidance and capabilities to secure federal civilian executive branch (FCEB) agencies’ cloud business application environments and protect federal information that is created, accessed, shared, and stored in those environments. 

> The CISA SCuBA SCBs for M365 help secure federal information assets stored within M365 cloud business application environments through consistent, effective, and manageable security configurations. CISA created baselines tailored to the federal government’s threats and risk tolerance with the knowledge that every organization has different threat models and risk tolerance. Non-governmental organizations may also find value in applying these baselines to reduce risks.

Reviewer: ```suggestion
The CISA SCuBA Security Configuration Baselines (SCB) for M365 help secure federal information assets stored within M365 cloud business application environments through consistent, effective, and manageable security configurations. CISA created baselines tailored to the federal government’s threats and risk tolerance with the knowledge that every organization has different threat models and risk tolerance. Non-governmental organizations may also find value in applying these baselines to reduce risks.
```

Author: added to first paragraph instead to match example.

#### MS.AAD.2.1v1
Users detected as high risk SHALL be blocked.

> <!--Policy: MS.AAD.2.1v1; Criticality: SHALL -->

Reviewer: See comments in other PR review.  Criticality needs to include additional info. 
```suggestion
<!--Policy: MS.AAD.2.1v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

- _Rationale:_ By blocking users determined as high risk, compromised accounts can be prevented from accessing the tenant.
- _Last modified:_ June 2023
- _Note:_ Users who are determined to be high risk by AAD Identity Protection can be blocked from accessing the system via an AAD Conditional Access policy. A high risk user will be blocked until an administrator remediates their account.

#### MS.AAD.2.2v1
A notification SHOULD be sent to the administrator when high-risk users are detected.

> <!--Policy: MS.AAD.2.2v1; Criticality: SHOULD -->

Reviewer: ```suggestion
<!--Policy: MS.AAD.2.2v1; Criticality: SHOULD/NOT-IMPLEMENTED -->
```

<!--Policy: MS.AAD.3.1v1; Criticality: SHALL -->
- _Rationale:_ The security risk of allowing weaker forms of MFA is that they do not protect against sophisticated phishing attacks. By enforcing methods which are resistant to phishing those risks are minimized.
- _Last modified:_ June 2023

#### MS.AAD.3.2v1
If phishing-resistant MFA has not been enforced yet, then an alternative MFA method SHALL be enforced for all users.

<!--Policy: MS.AAD.3.2v1; Criticality: SHALL -->
- _Rationale:_ This is a backup security policy to help protect the tenant in the event that phishing-resistant MFA has not been enforced yet. This policy requires that MFA is enforced and thus reduces the risks of single form authentication.
- _Last modified:_ June 2023
- _Note:_ If a conditional access policy has been created that enforces phishing-resistant MFA, then this policy is not necessary. This policy does not dictate the specific MFA method.

#### MS.AAD.3.3v1
If phishing-resistant MFA has not been enforced yet and Microsoft Authenticator is enabled, it SHALL be configured to show login context information.

> <!--Policy: MS.AAD.3.3v1; Criticality: SHALL -->

Reviewer: Rego has Should/Not-Implemented.  I see control reads as SHALL so I assume it should be SHALL/NOT-IMPLEMENTED.  Need issue for Rego?

- _Rationale:_ This is a backup security policy to help protect the tenant in the event that phishing-resistant MFA has not been enforced yet and Microsoft Authenticator is being used. This policy helps improve the security of Microsoft Authenticator by showing the user context information which helps reduce MFA phishing compromises.
- _Last modified:_ June 2023

#### MS.AAD.3.4v1
The Authentication Methods Manage Migration feature SHALL be set to Migration Complete.

<!--Policy: MS.AAD.3.4v1; Criticality: SHALL -->
- _Rationale:_ By configuring the Manage Migration feature to Migration Complete, we ensure that the tenant has disabled the legacy authentication methods screen. The MFA and SSPR authentication methods are both managed from a central admin page thereby reducing administrative complexity and
reducing the chances of security misconfigurations.
- _Last modified:_ June 2023

#### MS.AAD.3.5v1
The authentication methods SMS, Voice Call, and Email OTP SHALL be disabled.

> <!--Policy: MS.AAD.3.5v1; Criticality: SHALL -->

Reviewer: ```suggestion
<!--Policy: MS.AAD.3.5v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

#### MS.AAD.4.1v1
Security logs SHALL be sent to the agency's Security Operations Center for monitoring.

> <!--Policy: MS.AAD.4.1v1; Criticality: SHALL -->

Reviewer: ```suggestion
<!--Policy: MS.AAD.4.1v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

- _Last modified:_ June 2023

#### MS.AAD.5.2v1
Only administrators SHALL be allowed to consent to applications.

<!--Policy: MS.AAD.5.2v1; Criticality: SHALL -->
- _Rationale:_ Ensuring that only specific privileged users can consent to applications reduces the risks of users giving insecure applications access to their data via [consent grant attacks](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants?view=o365-worldwide).
- _Last modified:_ June 2023

#### MS.AAD.5.3v1
An admin consent workflow SHALL be configured for applications.

<!--Policy: MS.AAD.5.3v1; Criticality: SHALL -->
- _Rationale:_ Configuring an admin consent workflow helps support the risk reduction of the previous policy by setting up a process for users to securely request access to applications necessary for business purposes. Administrators get the opportunity to review the permissions requested by new applications and approve or deny access based on a risk assessment.
- _Last modified:_ June 2023

#### MS.AAD.5.4v1
Group owners SHALL NOT be allowed to consent to applications.

> <!--Policy: MS.AAD.5.3v1; Criticality: SHALL -->

Reviewer: ```suggestion
<!--Policy: MS.AAD.5.4v1; Criticality: SHALL -->
```
